index.mjs

```diff
@@ -56,7 +56,8 @@ export function registerGlobals() {
 }
 
 export function $(pieces, ...args) {
-  let {verbose, shell, prefix} = $
+  let {verbose, shell, prefix, spawn} = $
+  let {timeout, killSignal, maxBuffer = 200 * 1024 * 1024 /* 200 MiB*/} = spawn
```

Comment:
Let's remove nesting and put maxBuffer in $ as well.

Reply:
Ok. Done